untrusted/urts_sys/src/lib.rs

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) 2022 The MobileCoin Foundation
+// See https://download.01.org/intel-sgx/sgx-dcap/1.13/linux/docs/Intel_SGX_Enclave_Common_Loader_API_Reference.pdf
+//
+#![allow(non_upper_case_globals)]
+#![allow(non_camel_case_types)]
+#![allow(non_snake_case)]
```

Comment:
Optional/Equivalent: Merge these
```suggestion
#![allow(non_camel_case_types, non_snake_case, non_upper_case_globals)]
```